resources/views/layouts/layout.blade.php

```diff
@@ -31,26 +31,31 @@
 
         <div class="collapse navbar-collapse" id="main-navbar">
             <ul class="navbar-nav me-auto">
-            <li class="nav-item">
-                <a class="nav-link text-dark {{ (request()->segment(1) == 'index') ? 'font-weight-bold' : '' }}" href="/">Startpagina</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link text-dark {{ (request()->segment(1) == 'training') ? 'font-weight-bold' : '' }}" href="/training">Trainingen</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link text-dark {{ (request()->segment(1) == 'evenement') ? 'font-weight-bold' : '' }}" href="/evenement">Evenementen</a>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle text-dark" id="navbarDropdownActiviteiten" role="button" data-bs-toggle="dropdown"  aria-haspopup="true" aria-expanded="false">
```

Comment:
Als je ' tabindex="0" ' toevoegd in de <a> tag bij elke dropdown dan kun je met het keyboard de navigatiebalk navigeren dmv de tab key en de pijltjes.

Reply:
ik zal dit de volgende commit erin zetten